server/app/views/admin/questions/QuestionEditView.java

```diff
@@ -457,10 +468,94 @@ private DomContent buildUniversalQuestion(QuestionForm questionForm) {
             ViewUtils.makeToggleButton(
                 /* fieldName= */ "isUniversal",
                 /* enabled= */ questionForm.isUniversal(),
+                /* hidden= */ false,
                 /* idPrefix= */ Optional.of("universal"),
                 /* text= */ Optional.of("Set as a universal question")));
   }
 
+  private FieldsetTag buildPrimaryApplicantInfoSection(QuestionForm questionForm) {
+    FieldsetTag result =
+        fieldset()
+            .withId("primary-applicant-info")
+            .with(legend("Primary Applicant Information").withClass(BaseStyles.INPUT_LABEL));
+    PrimaryApplicantInfoTag.getAllTagsForQuestionType(questionForm.getQuestionType())
+        .forEach(
+            primaryApplicantInfoTag -> {
+              Optional<QuestionDefinition> currentQuestionForTag =
+                  questionService.getReadOnlyQuestionServiceSync().getUpToDateQuestions().stream()
+                      .filter(
+                          qd -> qd.getPrimaryApplicantInfoTags().contains(primaryApplicantInfoTag))
+                      .findFirst();
+              boolean differentQuestionHasTag =
+                  currentQuestionForTag
+                      .map(question -> !question.getName().equals(questionForm.getQuestionName()))
+                      .orElse(false);
+              DivTag tagSubsection = div().withClass("cf-primary-applicant-info-subsection");
+
+              if (differentQuestionHasTag) {
+                // Only include the alert. We'll swap the text based on the state of the
+                // universal toggle in admin_question_edit.ts.
+                String otherQuestionName =
+                    currentQuestionForTag.map(QuestionDefinition::getName).orElseThrow();
+                String alreadySetAlertText =
+                    String.format(
+                        "You cannot edit this setting since this property is already set on a"
```

Comment:
*very nit (phrasing)*: Here and on line 507 we use "since", but on line 508 we use "because". 

I think it'd read slightly cleaner if we used "because" everywhere, (or at least used the same conjunction for both).

Also "this property" refers to the PAI toggle, right? imo naming it is helpful, so "You cannot make this question a Primary Applicant Info question because the question named "%s" is already set as a Primary Applicant Info question." would be more clear.

Reply:
These suggestions look great @avaleske . Lets move ahead with them. @nb1701